components/public-api/gitpod/experimental/v1/projects.proto

```diff
@@ -47,6 +47,7 @@ message PrebuildSettings {
     bool keep_outdated_prebuilds_running = 2;
     bool use_previous_prebuilds = 3;
     int32 prebuild_every_nth = 4;
+    bool enable_prebuilds = 5;
```

Comment:
not sure it is necessary in this PR, but not so important

Reply:
it is! 
see we're using the server API to write project settings, and PAPI to read! if this is not mapped properly, it just doesn't work with dashboard. it's out of scope for this PR to resolve this situation, but we need to act on this. not driving to the end, means we're worsening the situation, i.e. instead of an improvement this is a severe degradation, as it slows us down for each feature work in this area.